server/config/config.go

```diff
@@ -209,6 +211,7 @@ func DefaultConfig() *Config {
 			PruningInterval:   "0",
 			MinRetainBlocks:   0,
 			IndexEvents:       make([]string, 0),
+			IAVLCacheSize:     50000000, // 50 MB
```

Comment:
I think this needs to be divided by 64, since I believe this cache size is actually the number of internal nodes. Each internal node IIRC should take 64 bytes, 32 for the left sibling, and 32 bytes for the right sibling.

Reply:
divided by 64.

Reply:
Ok I think we actually need to divide by 128 after looking at the struct defined here; https://github.com/cosmos/iavl/blob/75d6e9829a4a29b1b2fe02d0f2ead3dd7ae9388d/node.go#L17-L29

Still definitely glad we added this and current numbers are probs fine, just need to update the comments